app/assets/stylesheets/top_bar.scss

```diff
@@ -0,0 +1,38 @@
+.has-submenu > a {
+  border-left: 1px solid white;
+  margin-left: 1rem;
+}
+
+.spacer {
+  margin-bottom: rem-calc(50);
+}
+
+.top-bar {
+  background-color: $primary-color;
+}
+
+.top-bar a {
+  color: white;
+  background-color: $primary-color;
+  text-align: right;
+}
+
+.top-bar-right ul li {
+  background-color: $primary-color;
+  color : white;
+}
+
+.top-bar-title {
+  float: left;
+  padding: 0.2rem;
+
+  h5 {
+    margin: 0;
+  }
+}
+
+.top-bar-title a {
+  font-family: $title-font-family;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.